### DISCUSSION TITLE
The batch file wont launch

### DISCUSSION THREAD

**Discussion by kezioo69** [w-okada/voice-changer#1350] [Q&A]
I tried to open the start_http file but that didnt work so I tried the start_https which worked, and it finished downloading but the app didnt open so I closed the command prompt and tried to reopen it. Now both files will open for like half a second and then close on its own and I can't get anything to work.

**Comment by AnnoyingFox** [comment]
Decompress the main file again to a new folder placed in the root of a different drive than C (if you haven't more drives than C then use C) and try again. Your installation is probably corrupted as it failed to download the necessary files the first time.
You can also try to download the latest v1 and v2 versions here and see which one works better for you.

https://huggingface.co/wok000/vcclient000/tree/main

**Reply by kezioo69** [reply]
Same issue except it won't even start the download anymore, terminal will open for half a second and just close

**Comment by AnnoyingFox** [comment]
We really lack many info to further help you.
When you open an issue, there is a form to fill with the necessary info to have a better start but in this case it was omitted. We don't know which version of the software are you using, which GPU are you using, operating system, CPU, etc.

**Reply by kezioo69** [reply]
Can I ask where I can find the form? I couldn't find it. Also I am using the v.1.5.2.9e on a windows 11 laptop, core i-7 processor with 32gb ram, and a geforce mx 150.

**Comment by AnnoyingFox** [comment]
The form appears when you open a new issue.
Anyway, we can start by updating your version. I gave you a link with the most up to date versions of RVC 1 and also RVC 2.
I thought you would have already downloaded and tried them. If you keep using an old version, there is a higher chance that a bug that has been already fixed may keep affecting you. Please kindly update RVC1 or RVC2 to the latest versions. I'd recommend trying first v1 and if it doesn't work, then also try v2 since is newer.

Something to keep in mind, though, is that your computer is unlikely to be able to handle the software properly. You can give it a try in both, CPU and GPU modes and see if one of the two can perform well enough to not get an immense delay but is unlikely that you will get a good experience.
